[All Hosts] (auth) SSO not supported in 'Edit in Teams' (#4088)
OfficeDev/office-js-docs-pr: docs/develop/troubleshoot-sso-in-office-add-ins.md

There are several possible causes.

- The add-in is running on a platform that does not support the `getAccessToken` API. For example, it is not supported on iPad. See also [Identity API requirement sets](/javascript/api/requirement-sets/common/identity-api-requirement-sets).

> - The Office document was opened from the **Files** tab of a Teams channel using the **Edit in Teams** option on the **Open** drop down menu. The `getAccessToken` API is not supported in this scenario.

Reviewer: Per [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28800), this should be dropdown

```suggestion
- The Office document was opened from the **Files** tab of a Teams channel using the **Edit in Teams** option on the **Open** dropdown menu. The `getAccessToken` API isn't supported in this scenario.
```
